scripts/common_cmdline.sh

```diff
@@ -70,6 +70,7 @@ function compileFull
     local errors; errors=$(grep -v -E \
         -e 'Warning: This is a pre-release compiler version|Warning: Experimental features are turned on|pragma experimental ABIEncoderV2|^ +--> |^ +\||^[0-9]+ +\| ' \
         -e 'Warning: Yul is still experimental. Please use the output with care.' \
+        -e 'Assembler run successful, no output requested.' \
```

Comment:
```suggestion
        -e '^Assembler run successful, no output requested.$' \
```

Reply:
Actually, I'd add that to all of those excluded lines.

Reply:
Got it, added that to every one of those.